scripts/run-gradually-deprecated.sh

```diff
@@ -124,11 +130,27 @@ function runDeprecatedFunctions {
     echo "Please remove the references to 'd.SetId("") from the Data Sources listed below"
     echo "and raise an error instead:"
     echo ""
+    echo "$result"
+    exit 1
+  fi
+  result=$(grep -Ril "markasgone" ./internal/services/**/*.go | grep data_source)
```

Comment:
Same here
```suggestion
  result=$(grep -Ril "markasgone" ./internal/services/**/*data_source*.go)
```

Reply:
I started out with that but alas we have files in the codebase as `data_source_XXX` too - since we can't guarantee users will follow the naming convention, I think grepping for it it contains `data_source` is probably ok for now?

Reply:
The glob at either end of the name should match zero or more chars? It's not major anyway :)

Reply:
oh fair point, yeah good shout